### PR TITLE
JAVA SDK V1 Updates 7.31.18

### PR DESCRIPTION
**Pull request by jpvajda**
Includes:

New Get Organization endpoint
Multi-Tenant x-org-id documentation and functionality.
Corrections to to Command Endpoint Documentation
Remove ssh_keys from systemuserpost model
GET applications was referencing a generic model so its now pointed it to the correct one in: ApplicationlIst.

### CODE REVIEW

**Comment by jpvajda** [comment]
@ppg @adducci more conflicts... blah...

File: jcapiv1/docs/SystemusersApi.md [hunk 1]
```diff
@@ -141,7 +136,7 @@ Name | Type | Description  | Notes
 
 <a name="systemusersList"></a>
 # **systemusersList**
-> Systemuserslist systemusersList(contentType, accept, limit, skip, sort, fields, filter)
```
Comment (adducci) [inline jcapiv1/docs/SystemusersApi.md]: Just double checking that this was an expected change

Comment (jpvajda) [reply jcapiv1/docs/SystemusersApi.md]: limit / skip / sort aren't supported traits so I removed them from Docs.

Comment (adducci) [reply jcapiv1/docs/SystemusersApi.md]: This one removes `contentType` and `accept` from systemusers list

**Comment by jpvajda** [comment]
Going to close this and resubmit a fresh PR from a fresh branch to try to resolve these conflicts.

**Pull request closed without merging**